ui/components/app/detected-token/detected-token.js

```diff
@@ -82,7 +68,22 @@ const DetectedToken = ({ setShowDetectedTokens }) => {
         },
       });
     }
+
     if (deSelectedTokens.length < detectedTokens.length) {
+      selectedTokens.forEach((importedToken) => {
```

Comment:
NIT: I think we can improve the readability of this code by extracting the add token logic into its own function

```
      // const selectedTokens = ... 
      
      const tokenSymbols = selectedTokens.map(({ symbol }) => symbol);
      await dispatch(importTokens(selectedTokens));
      dispatch(setNewTokensImported(tokenSymbols.join(', ')));

      // trackEvent logic
```